### PR TITLE
Add Guess My Next Trip Function

### PR DESCRIPTION
**Pull request by obieglok**
Added Guess My Next Trip Function to the Travel Page in order to incorporate some Javascript into my portfolio.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,16 @@
...
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function guessMyTrip()
+{
+  const places=["thailand", "singapore", "tokyo", "australia","peru", "hawaii", "vietnam"];
+  var input =document.getElementById("user_input").value;
+  console.log(input);
+  if(places.includes(input.toLowerCase())){
+      document.getElementById('display').innerHTML = "Well done you guessed it correctly! " + input +" is one of the places I will be travelling to on my Round the World Trip in 2022!";
```
Comment (marvel308) [inline portfolio/src/main/webapp/script.js:23]: column length > 80 which isn't allowed by style guide https://google.github.io/styleguide/jsguide.html#formatting-column-limit. Is there a way to make this multi line?
Example 
var x = "a" +
"b" +
"c"

Comment (marvel308) [reply portfolio/src/main/webapp/script.js:23]: Some ideas http://go/js-style-external#formatting-line-wrapping

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function guessMyTrip()
+{
+  const places=["thailand", "singapore", "tokyo", "australia","peru", "hawaii", "vietnam"];
+  var input =document.getElementById("user_input").value;
```
Comment (marvel308) [inline portfolio/src/main/webapp/script.js]: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -15,14 +15,16 @@
...
+  console.log(input);
```
Comment (marvel308) [inline portfolio/src/main/webapp/script.js:21]: Is this to debug or do you want this to print every time?

Comment (obieglok) [reply portfolio/src/main/webapp/script.js:23]: Just for debug,  I will remove it now.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (obieglok) [reply portfolio/src/main/webapp/script.js]: I will change var to let now

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (obieglok) [reply portfolio/src/main/webapp/script.js]: thanks ,changed it

**Pull request merged**